### PR TITLE
DOCSUP-11656:  Document the function_range_max_elements_in_block setting

### PR DESCRIPTION
**Pull request by ka1bi4**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Documentation for #26305.

Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
- Add the `function_range_max_elements_in_block` setting and update `range` function details.

### CODE REVIEW

File: docs/en/operations/settings/settings.md [hunk 1]
```diff
@@ -109,6 +109,21 @@ Enables or disables [fsync](http://pubs.opengroup.org/onlinepubs/9699919799/func
...
 
+## function_range_max_elements_in_block {#settings-function_range_max_elements_in_block}
+
+Sets the safety threshold for data volume generated by function [range](../../sql-reference/functions/array-functions.md#range). Defines the maximum number of values generated by function per block of data (sum of array sizes for every row in a block).
+
+Possible values:
+
+-   Positive integer.
+
+Default value: `500000000`.
```
Comment (olgarev) [inline docs/en/operations/settings/settings.md:120]:
```suggestion
Default value: `500,000,000`.
```

File: docs/ru/operations/settings/settings.md [hunk 2]
```diff
@@ -85,6 +85,21 @@ ClickHouse применяет настройку в тех случаях, ко
 
 Имеет смысл выключать, если на сервере миллионы мелких таблиц-чанков, которые постоянно создаются и уничтожаются.
 
+## function_range_max_elements_in_block {#settings-function_range_max_elements_in_block}
+
+Устанавливает порог безопасности для объема данных, создаваемого функцией [range](../../sql-reference/functions/array-functions.md#range). Задаётся максимальное количество значений, генерируемых функцией на блок данных (сумма размеров массива для каждой строки в блоке).
```
Comment (olgarev) [inline docs/ru/operations/settings/settings.md:90]:
```suggestion
Устанавливает порог безопасности для объема данных, создаваемого функцией [range](../../sql-reference/functions/array-functions.md#range). Задаёт максимальное количество значений, генерируемых функцией на блок данных (сумма размеров массивов для каждой строки в блоке).
```

File: docs/ru/operations/settings/settings.md [hunk 3]
```diff
@@ -85,6 +85,21 @@ ClickHouse применяет настройку в тех случаях, ко
...
+
+Возможные значения:
+
+-   Положительное целое.
+
+Значение по умолчанию: `500000000`.
```
Comment (olgarev) [inline docs/ru/operations/settings/settings.md:96]:
```suggestion
Значение по умолчанию: `500 000 000`.
```

File: docs/en/operations/settings/settings.md [hunk 4]
```diff
@@ -109,6 +109,21 @@ Enables or disables [fsync](http://pubs.opengroup.org/onlinepubs/9699919799/func
...
+
+Sets the safety threshold for data volume generated by function [range](../../sql-reference/functions/array-functions.md#range). Defines the maximum number of values generated by function per block of data (sum of array sizes for every row in a block).
+
+Possible values:
+
+-   Positive integer.
+
+Default value: `500,000,000`.
+
+**See Also**
```
Comment (mathalex) [inline docs/en/operations/settings/settings.md:122]:
```suggestion
See also:
```

File: docs/ru/operations/settings/settings.md [hunk 5]
```diff
@@ -85,6 +85,21 @@ ClickHouse применяет настройку в тех случаях, ко
...
+
+Устанавливает порог безопасности для объема данных, создаваемого функцией [range](../../sql-reference/functions/array-functions.md#range). Задаёт максимальное количество значений, генерируемых функцией на блок данных (сумма размеров массивов для каждой строки в блоке).
+
+Возможные значения:
+
+-   Положительное целое.
+
+Значение по умолчанию: `500 000 000`.
+
+**См. также**
```
Comment (mathalex) [inline docs/ru/operations/settings/settings.md:98]:
```suggestion
См. также:
```

File: docs/ru/operations/settings/settings.md [hunk 6]
```diff
@@ -85,6 +85,21 @@ ClickHouse применяет настройку в тех случаях, ко
 
 Имеет смысл выключать, если на сервере миллионы мелких таблиц-чанков, которые постоянно создаются и уничтожаются.
 
+## function_range_max_elements_in_block {#settings-function_range_max_elements_in_block}
+
+Устанавливает порог безопасности для объема данных, создаваемого функцией [range](../../sql-reference/functions/array-functions.md#range). Задаёт максимальное количество значений, генерируемых функцией на блок данных (сумма размеров массивов для каждой строки в блоке).
+
+Возможные значения:
+
+-   Положительное целое.
```
Comment (mathalex) [inline docs/ru/operations/settings/settings.md:94]:
```suggestion
-   Положительное целое число.
```

**Review by mathalex — COMMENTED** [review]
For the file consistency (there are several more places in the file, which also should be fixed in the future for uniformity).

File: docs/en/operations/settings/settings.md (hunk 4 above)
Comment (ka1bi4) [reply docs/en/operations/settings/settings.md:122]: Hi, Alexey!
In the ClickHouse style guide, we have description templates (for example [setting template] (https://github.com/ClickHouse/ClickHouse/blob/master/docs/_description_templates/template-setting.md)) so I strictly follow the agreed syntax.

Comment (mathalex) [reply docs/en/operations/settings/settings.md:122]: Yes, I have already acknowledged it. Thanks anyway.

File: docs/ru/operations/settings/settings.md (hunk 6 above)
Comment (mathalex) [reply docs/ru/operations/settings/settings.md:94]: Anyway, one suggestion here (out of three) is not about style (about translating the word 'integer') -- please confirm.

**Review by mathalex — APPROVED** [review]

**Pull request merged**